src/components/molecules/UserList/UserList.tsx

```diff
@@ -44,7 +44,9 @@ const UserList: React.FunctionComponent<PropsType> = ({
   const [selectedUserProfile, setSelectedUserProfile] = useState<
     WithId<User>
   >();
-  users = users?.filter((user) => !user.anonMode && user.partyName && user.id); // quick fix to get rid of anonymous users
+  const users = userList?.filter(
+    (user) => !user.anonMode && user.partyName && user.id
+  ); // quick fix to get rid of anonymous users
```

Comment:
Is this still counted as a 'quick fix'? We're also removing the anonMode users.. is that what we want to be doing? And if so, what makes this a 'quick fix' rather than just our intended future functionality here?